src/commands/shared.ts

```diff
@@ -165,11 +174,23 @@ export const provisionRequest = async (
 };
 
 export const requestToSsh = (request: AwsSsh): SshRequest => {
```

Comment:
I wrote this because I didn't like that `generated.name` was the `role.name` and to flatten out the object so that if the `permission` changes, we only need to change the `sshRequest` in one place (here where it's getting transformed). Probably isn't all that necessary to be honest.